__tests__/test.js

```diff
@@ -89,9 +89,35 @@ describe('git-repo-is-up-to-date', () => {
     const newFilePath = path.join(clonedRepoPath, 'tmp.js')
     await fs.writeFile(newFilePath, 'blah')
 
+    // debug git status
+    await execa('git', ['status'], { cwd: clonedRepoPath })
+
     // add file
     await execa('git', ['add', newFilePath], { cwd: clonedRepoPath })
 
+    // debug git status
+    await execa('git', ['status'], { cwd: clonedRepoPath })
```

Comment:
These `git status` calls don't seem to be output to the GitHub Actions log. Is that intentional? Are they still needed? It seems like there was a bit of churn to figure out what was happening with the test -- is there any reason not to log more things to the console during the tests to increase visibility of what's happening?

Reply:
Could the [answer found here](https://stackoverflow.com/a/61112017) help print the console?

Reply:
whoops, definitely not needed anymore. Removed in 0e1ef61b87da99a059e8f660e536eec3b83ad8db.